src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: How do I register an Ethereum account?
+description: A step-by-step guide on Ethereum account creation using a wallet.
+lang: en
+---
+
+# How to: "Register" an Ethereum account
+
+Anyone can create an Ethereum account for free with a specific type of app commonly known as a wallet. Wallets hold the keys that let you hold, send and receive crypto. You can also connect to projects on Ethereum that will let you trade NFTs, exchange coins, access play-to-earn games, unlock Defi and much more.
+
+## Step 1: Browse our list of wallets
+
+A wallet is like an online bank account for Ethereum. There are dozens of different wallets to choose from—mobile, desktop, or even browser extensions. Our curated list of trusted wallets is a good place to start. 
+
+<ButtonLink to="/wallets/find-wallet/">
+  Find a wallet
+</ButtonLink>
+
+## Step 2: Choose a wallet that suits your needs
+
+If you are new, you can activate the “New to crypto” filter to view only those wallets that include all necessary features we think would be particularly suitable for beginners. There are also other profile filters to cater to your needs.
+
+## Step 3: Download and install your wallet app
+
+Once you have decided on a specific wallet, visit their website or app store, download and install it. All of them should be free. 
+
+## Step 4: Open the app and generate or import your Ethereum account
+
+The first time you open your new wallet you might be asked to choose between creating a new account or importing an existing one. Click on the new account creation. 
+
+
+## Step 5: Store your recovery phrase
+
+Some apps will request you to save a secret recovery phrase. Keeping this secret recovery phrase safe is extremely important! Any person who knows this secret recovery phrase can take control of all your accounts generated by these words. Never share this with anyone. This phrase should contain 12 to 24 randomly generated words (the order of the words matters).
+
+Once you have saved your recovery phrase you should see your wallet dashboard with your balance. Check out our guide: [how to use a wallet.](/guides/how-to-use-a-wallet)
+
+<InfoBanner shouldSpaceBetween emoji=":eyes:">
+  <div>Want to learn more?</div>
+  <ButtonLink to="/guides/">
+    See our other guides
+  </ButtonLink>
+</InfoBanner>
+
+### Frequently asked questions
+
+### Are my wallet and my Ethereum account the same?
+No, wallets have the ability to create a new Ethereum account, but the account lives on Ethereum and wallets connect to it. So if you lose your wallet you can still recover your account with the recovery phrase.
+
+### Can I send bitcoin to an Ethereum address, or ether to a Bitcoin address?
+
+No you cannot, you need two different accounts and receiving addresses with a different format altogether. There are many EVM (Ethereum Virtual Machine) compatible tokens & blockchains that you can use with your Ethereum address.
```

Comment:
Understand the need for simplicity but propose to be more explicit about the difference in protocols. This will better guide the reader to understand the underlying reason for incompatibility.

Proposed:
- No you cannot as bitcoin and ether exists on two separate networks (i.e. different blockchains), each with their own bookkeeping models and address formats. There are many EVM ([Ethereum Virtual Machine](/developers/docs/evm/)) compatible tokens & blockchains that you can use with your Ethereum address.

Reply:
+1

Reply:
changed